src/firefly/js/api/ApiUtil.js

```diff
@@ -0,0 +1,97 @@
+
+/*
+ * License information at https://github.com/Caltech-IPAC/firefly/blob/master/License.txt
+ */
+
+import React, {Component} from 'react';
+import ReactDOM from 'react-dom';
+import {get,isEmpty} from 'lodash';
+import {isElement,isString} from 'lodash';
+import {logError} from '../util/WebUtil.js';
+import {dispatchAddSaga} from '../core/MasterSaga.js';
+import {take} from 'redux-saga/effects';
+
+// NOTE 
+// NOTE 
+//------------------------------------------------------------------------------------------
+// Anything that is exported here becomes part of the lowlevel API
+// It should have a jsdoc
+//------------------------------------------------------------------------------------------
+// NOTE 
+// NOTE 
+
+/**
+ * Is in debug mode
+ */
+export const isDebug = () => get(window, 'firefly.debug', false);
+
+/**
+ * show a debug message if debugging is enabled
+ * @param msg any number of messages
+ */
+export function debugMsg(...msg) {
+    if (!isEmpty(msg) && isDebug()) {
+        logError(...msg);
+    }
+}
+
+
+
+/**
+ *
+ * @param {string|{}} div a div element or a string id of the div element
+ * @param {{}} Component a react component
+ * @param {{}} [props] props for the react component
+ * @param {{}} [style] css style object, style of this root element
+ */
+export function renderDOM(div, Component, props, style={width:'100%', height:'100%'}) {
+    const divElement= isString(div) ? document.getElementById(div) : div;
+
+    if (!isElement(divElement)) debugMsg(`the div element ${isString(div)?div:''} is not defined in the html` );
+    if (!Component) debugMsg('Component must be defined');
+
+
+    const renderStuff= (
+        <div style={style}
+             className='rootStyle' >
+            <Component {...props} />
+        </div>
+    );
```

Comment:
I wonder if we can simply add rootStyle directly into the divElement instead of creating a new one.
It's more natural for the api user to style their div directly instead of passing it into this function.